### PR TITLE
feat: add jumpstart send amount screen

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

This PR adds the jumpstart enter amount screen.
- add a selector to get jumpstart tokens
- add the new screen
- add a hook to prepare the jumpstart transaction
- add tests

### Test plan

https://github.com/valora-inc/wallet/assets/20150449/ddca2609-d62d-40e0-bf8a-85c87ec73bad

### Related issues

- Relates to RET-993

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [x] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: src/send/JumpstartEnterAmount.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    //    saga handle it.
+    // 2. Generate the link and pass the link in a navigation parameter to the
+    //    next screen. (use navigateClearingStack so that the user cannot come
+    //    back to this screen and reuse the private key)
+    // 3. add analytics
+  }
+
+  const prepareJumpstartTransactions = usePrepareJumpstartTransactions()
+
+  const handlRefreshPreparedTransactions = (
```
Comment (bakoushin) [inline src/send/JumpstartEnterAmount.tsx:39]: typo

**Review by bakoushin — APPROVED** [review]
🚀 Epic work!

I assume that `EnterAmount` and `SendEnterAmount` haven't changed and will converge with the `test-send-abstract-1`

**Comment by bakoushin** [comment]
should we group all jumpstart-related files into a dedicated directory within `send`?

**Comment by kathaypacific** [comment]
> 🚀 Epic work!
> 
> I assume that `EnterAmount` and `SendEnterAmount` haven't changed and will converge with the `test-send-abstract-1`

yes sorry i didn't realise this branch had a conflict so the diffs must have shown more than there was. there is no change to `EnterAmount` and `SendEnterAmount` in this PR

**Comment by kathaypacific** [comment]
> should we group all jumpstart-related files into a dedicated directory within `send`?

yes that's a great suggestion! i ended up grouping them in the existing `jumpstart` folder, rather than having both that and `send/jumpstart`, hope that still addresses the essence of your suggestion!

**Pull request merged**